src/Genetics.jl

```diff
@@ -1,68 +1,68 @@
-# using Diversity
-# using Diversity.API
+using Diversity
+using Diversity.API
 
-# using PopGen
-# using StringDistances
-# using LinearAlgebra
+using PopGen
+using StringDistances
+using LinearAlgebra
 
-# using FASTX
+using FASTX
 
-# abstract type AbstractGeneticTypes{PopData} <: 
-#     Diversity.API.AbstractTypes
-# end
+abstract type AbstractGeneticTypes{PopData} <: 
+    Diversity.API.AbstractTypes
+end
 
-# struct GeneticType{PopData} <: AbstractGeneticTypes{PopData}
-#     dat::PopData
-#     ntypes::Int64
-#     Zmatrix::Matrix{Float64}
-# end
+struct GeneticType{PopData} <: AbstractGeneticTypes{PopData}
+    dat::PopData
```

Comment:
So, as a for instance, I would make this some arbitrary "`Data`" type, which is defined when you construct the object (note `Data` must **not** already exist!):
```suggestion
struct GeneticType{Data} <: AbstractGeneticTypes{Data}
    dat::Data
```

Reply:
Alrighty, got both constructors working. Wondering though if `AbstractGenetic` is necessary? Since the two genetic types could just as easily be subtypes of `Diversity.API.AbstractTypes`, unless you want genetic types to be treated differently?